### ISSUE TITLE
@nrwl/web:rollup produces the wrong output

### ISSUE THREAD

**Issue by NetanelBasal** [nrwl/nx#8475]
## Current Behavior
When using the following configuration:

```json
    "build": {
      "executor": "@nrwl/web:rollup",
      "outputs": [
        "{options.outputPath}"
      ],
      "options": {
        "outputPath": "dist/packages/falso",
        "tsConfig": "packages/falso/tsconfig.lib.json",
        "project": "packages/falso/package.json",
        "entryFile": "packages/falso/src/index.ts",
        "format": [
          "esm",
          "cjs"
        ]
      }
    },
```

The build output in `package.json` is:

```json
  "main": "./index.umd.js",
  "module": "./index.esm.js",
  "typings": "./index.d.ts"
```

## Expected Behavior

```json
  "main": "./index.cjs.js",
  "module": "./index.esm.js",
  "typings": "./index.d.ts"
```

**Comment by NetanelBasal** [comment]
<img width="1059" alt="Screen Shot 2022-01-11 at 22 18 41" src="https://user-images.githubusercontent.com/6745730/149015136-8af55050-b73e-44d9-b7c6-ffd54df436c7.png">

The issue is here. It doesn't take into account the provided `format`.

**Comment by jaysoo** [comment]
This is fixed in latest Nx version (14.1.9)
